community/libplist/APKBUILD

```diff
@@ -1,35 +1,34 @@
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libplist
 pkgver=2.0.0
-pkgrel=1
+pkgrel=2
 pkgdesc="A library to handle Apple Property List format whereas it's binary or XML"
 url="http://libimobiledevice.org/"
 arch="all"
-license="GPL-2.0 LGPL-2.1"
-depends=""
-depends_dev="libxml2-dev glib-dev"
-makedepends="py2-setuptools swig python2-dev $depends_dev python2 cython2"
-subpackages="$pkgname-dev py-$pkgname:py $pkgname-util $pkgname++:cxx"
+license="GPL-2.0-or-later LGPL-2.1-or-later"
+makedepends="py2-setuptools swig python2-dev glib-dev cython"
+subpackages="$pkgname-static $pkgname-dev py-$pkgname:py $pkgname-util $pkgname++:cxx"
 source="http://www.libimobiledevice.org/downloads/libplist-$pkgver.tar.bz2"
 
-_builddir="$srcdir"/build
-_src="$srcdir"/$pkgname-$pkgver
 build() {
-	cd "$srcdir"
-	mkdir -p "$_builddir"
-	cd "$_builddir"
-	ac_cv_path_CYTHON=cython2 $_src/configure \
+	./configure \
 		--build=$CBUILD \
 		--host=$CHOST \
 		--prefix=/usr \
-		--mandir=/usr/share/man \
-		|| return 1
-	make || return 1
+		--mandir=/usr/share/man
+	make
+}
+
+check() {
+	# https://github.com/libimobiledevice/libplist/issues/118
+	if [ "$CARCH" != armv7 ]; then
+		make check -j1 # https://github.com/libimobiledevice/libimobiledevice/issues/777
+	fi
```

Comment:
This could switch to using `case` too.

Reply:
The rational is to make modifications easier.

```
case "${CARCH}" in (armv7) return 0 ;; esac # skip matches
```

Adding another architecture to skip is easier and doesn't require thinking through the boolean logic.